### ISSUE TITLE
2.1.0-preview2-26406-04 and xunit 2.3.1 result in Microsoft.Extensions.Logging.Abstractions not found

### ISSUE THREAD

**Issue by jsacapdev** [xunit/xunit#1692]
## Steps to reproduce the issue
1. `git clone https://github.com/jsacapdev/k8-ci-cd.git`
2. `cd .\test\KEight.Api.Tests\`
3. `dotnet xunit -c Release`

## Expected behavior

Tests run through to completion

## Actual behavior

```
Detecting target frameworks in KEight.Api.Tests.csproj...
Building for framework netcoreapp2.1...
  You are working with a preview version of the .NET Core SDK. You can define the SDK version via a global.json file in the current project.
 More at https://go.microsoft.com/fwlink/?linkid=869452
  You are working with a preview version of the .NET Core SDK. You can define the SDK version via a global.json file in the current project.
 More at https://go.microsoft.com/fwlink/?linkid=869452
  KEight.Data -> C:\git\k8-ci-cd\src\KEight.Data\bin\Release\netstandard2.0\KEight.Data.dll
  KEight.Api -> C:\git\k8-ci-cd\src\KEight.Api\bin\Release\netcoreapp2.1\KEight.Api.dll
  KEight.Api.Tests -> C:\git\k8-ci-cd\test\KEight.Api.Tests\bin\Release\netcoreapp2.1\KEight.Api.Tests.dll
Running .NET Core 2.1.0-preview2-26406-04 tests for framework netcoreapp2.1...
xUnit.net Console Runner (64-bit .NET Core 4.6.26406.07)
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Extensions.Logging.Abstractions, Version=2.1.0.0, Culture=neutra
l, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Extensions.Logging.Abstractions, Version=2.1.0.0, Culture=neutra
l, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.
```

## Additional information (e.g. issue happens only occasionally)

`dotnet test -c Release` works succesfully

## Output of `dotnet --info`

```
.NET Core SDK (reflecting any global.json):
 Version:   2.1.300-preview2-008530
 Commit:    822ae6d43a

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.300-preview2-008530\

Host (useful for support):
  Version: 2.1.0-preview2-26406-04
  Commit:  6833f3026b

.NET Core SDKs installed:
  2.0.2 [C:\Program Files\dotnet\sdk]
  2.0.3 [C:\Program Files\dotnet\sdk]
  2.1.2 [C:\Program Files\dotnet\sdk]
  2.1.4 [C:\Program Files\dotnet\sdk]
  2.1.101 [C:\Program Files\dotnet\sdk]
  2.1.102 [C:\Program Files\dotnet\sdk]
  2.1.103 [C:\Program Files\dotnet\sdk]
  2.1.104 [C:\Program Files\dotnet\sdk]
  2.1.300-preview1-008174 [C:\Program Files\dotnet\sdk]
  2.1.300-preview2-008530 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.0-preview1-final [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.0-preview2-final [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.0-preview1-final [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.0-preview2-final [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.0-preview1-26216-03 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.0-preview2-26406-04 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by p3root** [comment]
same here, any updates?

**Comment by bradwilson** [comment]
Dup of and/or fixed by #1675
